distributed/scheduler.py

```diff
@@ -3279,7 +3279,11 @@ def __init__(
         self.log = deque(
             maxlen=dask.config.get("distributed.scheduler.transition-log-length")
         )
-        self.events = defaultdict(lambda: deque(maxlen=100000))
+        self.events = defaultdict(
+            lambda: deque(
+                maxlen=dask.config.get("distributed.scheduler.transition-log-length")
```

Comment:
Since the transition and events logging systems are independent of one another and store different types of data, users may want their corresponding `deque`s to have different maximum lengths. We could accomplish this by adding a separate `distributed.scheduler.events-log-length` config value to accompany the existing `distributed.scheduler.transition-log-length` value.

Reply:
Sounds reasonable to me. However, the idea was to provide the user with the possibility to avoid the OOM issue and I assumed somebody tweaking the `transition-log-length` settings would probably also want to reduce the overall memory footprint. I totally agree, that it makes sense to provide the possibility to configure these settings individually, but I would indeed prefer to keep this PR as is and add `distributed.scheduler.events-log-length` in a follow-up PR, if that is okay for you.